src/ui/ResponsiveComponents/ResponsiveComponentsManager.ts

```diff
@@ -124,7 +124,7 @@ export class ResponsiveComponentsManager {
     this.searchBoxElement = this.getSearchBoxElement();
     this.logger = new Logger(this);
     this.resizeListener = debounce(() => {
-      if (this.coveoRoot.width() != 0) {
+      if (this.coveoRoot.width() != 0 || this.searchInterface.options.responsiveMode !== 'auto') {
```

Comment:
I'd prefer the condition to be extracted in a named function that expresses what it's trying to do versus a long check.

`if (this.isAbleToDetermineMode())`, or something like that ?

Reply:
Makes sense to me as a similar check is done elsewhere.